workbench-core/datasets/src/ddbDataSetMetadataPlugin.ts

```diff
@@ -83,6 +83,10 @@ export class DdbDataSetMetadataPlugin implements DataSetMetadataPlugin {
     return dataSet;
   }
 
+  public async removeDataSet(dataSetId: string): Promise<void> {
+    await this._removeDataSetFromDdb(dataSetId);
```

Comment:
do we need to create a separate removeDataSetFromDdb when this exists within ddbDataSetMetadataPlugin

Reply:
removed